pyexchange/tethfinex.py

```diff
@@ -294,8 +303,8 @@ def get_trades(self, pair: str, page_number: int = 1) -> List[Trade]:
 
         result = self._get_orders("/trustless/v1/r/orders/hist")
 
-        executed_orders = filter(lambda order: 'EXECUTED' in order['status'], result)
-
+        #executed_orders = filter(lambda order: 'EXECUTED' in order['status'], result)
```

Comment:
will the result return always orders in EXECUTED state (that is trades)? If so you pls remove commented line

Reply:
Some trades will be 'PARTIALLY FILLED' instead of 'EXECUTED', but should still be returned by get_trades in my opinion

Reply:
Are there only  'PARTIALLY FILLED' and 'EXECUTED' orders returned by orders/hist endpoint? If I correctly recall canceled orders were returned too, hence we have to filter only filled orders in result

Reply:
Yes, filter is required because there are also Canceled orders (and live/open orders outside the 'hist' endpoint)

Reply:
I skipped the filter during the test because i only created orders deleted. I add the filter again.